docs/jade-getting-started.md

```diff
@@ -289,17 +292,21 @@ to build `jade-data-repo`.
 
 * You will need to run `render-configs.sh` before running integration tests.
 
-* Certain environment variables need to be set beforehand. Instances of `zzz`
-should be replaced by your initials or the environment (i.e. `dev`):
+Certain environment variables need to be set beforehand. Instances of `zzz`
+should be replaced by your initials:
 
 ```
 export VAULT_ADDR=https://clotho.broadinstitute.org:8200
 export PROXY_URL=https://jade-zzz.datarepo-dev.broadinstitute.org
 export GOOGLE_CLOUD_PROJECT=broad-jade-zzz
 export GOOGLE_APPLICATION_CREDENTIALS=/tmp/jade-dev-account.json
 export GOOGLE_SA_CERT=/tmp/jade-dev-account.pem
-export GOOGLE_CLOUD_DATA_PROJECT=broad-jade-zzz-data
+export GOOGLE_CLOUD_DATA_PROJECT=broad-jade-dev-data
```

Comment:
This should be your initials - It's important to not point to broad-jade-dev-data!

Reply:
@fboulnois you had mentioned this should be dev, so should all the 'zzz' just be initials then?

Reply:
GOOGLE_CLOUD_PROJECT can be set to broad-jade-dev or broad-jade-ZZ, but the data project (GOOGLE_CLOUD_**DATA**_PROJECT) needs to be set to your personal dev project.

Reply:
This is for a couple of reasons:
1. Unfortunately, our actual dev environment has been treated way too much like real environment and stores data that clients are relying on (but, now that we're on production, it should be moved to prod soon!). The actual data (so data in Big Query and Firestore) for these projects is stored in our "data" google projects. We don't want our test data mingling with this "real" data, or for anyone to accidentally delete real data. 
2. On each test run, we generate policies and these are linked to policies in IAM. We are limited to only 250 policies per project. So, if you run tests against broad-jade-dev-data, then we'll hit the IAM policy limit and then dev becomes unusable. We can clean out the test policies, but it's not a fun process.

Reply:
Yes, seconding what @snf2ye mentioned.